library/core/src/macros/panic.md

```diff
@@ -24,11 +24,11 @@ The multi-argument form of this macro panics with a string and has the
 
 See also the macro [`compile_error!`], for raising errors during compilation.
 
-[runwrap]: ../std/result/enum.Result.html#method.unwrap
-[`Option`]: ../std/option/enum.Option.html#method.unwrap
-[`Result`]: ../std/result/enum.Result.html
+[ounwrap]: Option::unwrap
+[runwrap]: Result::unwrap
```

Comment:
Why not just make the original version `Option::unwrap` and `Result::unwrap`?

Reply:
Let's make that follow-up PR, this one's been through enough 😆

Reply:
I forgot my the last time I commented on a PR that needs a follow up PR, I wanted to go back to that but I forgot which PR is it.

Reply:
Try one of these: https://github.com/rust-lang/rust/pulls?q=is%3Apr+reviewed-by%3Apickfire+label%3AT-doc+is%3Aclosed+label%3AA-intra-doc-links